pyproject.toml

```diff
@@ -118,20 +118,15 @@ replace = "[{new_version}] {now:%Y-%m-%d}"
 # ============================================================================
 
 [tool.black]
-# we should set this to 179
-line-length = 120
+line-length = 179
 
 [tool.ruff]
-# we should set this to 179
-line-length = 120
+line-length = 179
 indent-width = 4
 target-version = "py39"
 
 [tool.ruff.lint]
-# we should add "I"
-select = ["E", "F"]
-# we should remove this
-ignore = ["E501"]
+select = ["E", "F", "I"]
```

Comment:
Without having to read the ruff docs, does this imply we enable all? What is excluded? Maybe drop a line indicating why/what is done here

Reply:
there is also "B", "Q", ... which are things we don't need/want